APMrover2/system.cpp

```diff
@@ -145,7 +145,8 @@ void Rover::init_ardupilot()
     startup_ground();
 
     Mode *initial_mode = mode_from_mode_num((enum Mode::Number)g.initial_mode.get());
-    if (initial_mode == nullptr) {
```

Comment:
normally we ask that formatting changes be put into a separate commit.  This one to move the bracket to the next line doesn't actually fit with our coding convention (there's a page on the developer wiki) so we should back out this change.